### DISCUSSION TITLE
How do you enable the matterbridge-shelly plugin when using the matterbridge-home-assistant-addon?

### DISCUSSION THREAD

**Discussion by quintesse** [Luligu/matterbridge-shelly#64] [Q&A] [answered]
Just what the title says. I've gone through the docs and the Matterbridge UI but I couldn't find it

**Comment by Luligu** [comment, marked as answer]
Hi,
I answered you also in https://github.com/Luligu/matterbridge-home-assistant-addon/issues/2.

You mean how to install shelly plugin when you run matterbridge using the matterbridge-home-assistant-addon?

Just open the frontend clicking on:
![image](https://github.com/user-attachments/assets/ddf358c4-e723-47cd-9977-84505228f65b)

and install the plugin.

![image](https://github.com/user-attachments/assets/15c1d2cd-ff7d-444f-84df-23564de57b8f)

**Comment by quintesse** [comment]
Ok! Thanks, I completely missed those options.
